Index: src/validators/carValidator.js
===================================================================
diff --git a/src/validators/carValidator.js b/src/validators/carValidator.js
--- a/src/validators/carValidator.js	
+++ b/src/validators/carValidator.js	
@@ -1,7 +1,9 @@
 import Joi from "joi";
 
 export const carValidator = Joi.object({
-        brand: Joi.string().regex(/^[a-zA-Zа-яА-яёЁіІїЇ]{1,20}$/).required(),
+        brand: Joi.string().regex(/^[a-zA-Zа-яА-яёЁіІїЇ]{1,20}$/).required().messages({
+                'string.pattern.base':'username can only contain letters and must be between 1 and 20 characters',
+        }),
         price: Joi.number().min(0).max(1000000).required(),
         year: Joi.number().min(1990).max(new Date().getFullYear()).required()
 });
